src/ui/verificationSymbolStatusView.ts

```diff
@@ -352,7 +352,7 @@ export default class VerificationSymbolStatusView {
   }
 
   public getVerifiableRanges(uriString: string): Promise<Range[]> {
-    return this.getVerifiableRangesPromise(uriString).promise;
+    return this.getVerifiableRangesPromise(Uri.parse(uriString).toString()).promise;
```

Comment:
Here I would also prefer it if the uriString was properly encoded on the caller side.

Reply:
The encoding of VSCode is not the same as the encoding on the LSP for a specific case, so I don't know how else to do this.